docs/zagg_client_checks.asciidoc

```diff
@@ -0,0 +1,118 @@
+== Writing ZaggClient Checks
+
+This document is intended to be used as a reference to write checks in Openshift V3 using the Zagg interface.
+
+*Note:* This document assumes you are using the https://github.com/openshift/openshift-tools/tree/master/docker/oso-rhel7-zagg-client[oso-rhel7-zagg-client] docker container.  This container includes all the required libraries to communicate with a Zagg server.
+
+==== A Note About ZaggClient Checks
+
+Let's take a quick review and understanding of Zagg and Zabbix to clarify what is meant by 'ZaggClient checks'.
+
+ZaggClient checks are simple gathering of Zabbix items and sending them to the ZaggServer (which eventually sends them to a Zabbix Server).
+
+
+=== Zagg Clients Overview
+
+Currently there are two different methods that can be used to send data to Zagg.
+
+. *ZaggSender python libraries*:  This is the *PREFERRED* way to write checks. ZaggSender is a python interface used to send data to Zagg
+
+. *ops-zagg-client*:  This is a command line tool that can be used to send data to Zagg.
+
+Both of these clients read their default settings from the config file: `/etc/openshift_tools/zagg_client.yaml`.
+
+More information will be provided below on each of these methods
```

Comment:
no need for this line, I'm assuming people won't stop here and say "wtf, where is the rest" :)